medical-clinic/src/main/java/com/Mafiuz04/medicalclinic/Patient.java

```diff
@@ -0,0 +1,92 @@
+package com.Mafiuz04.medicalclinic;
+
+import java.time.LocalDate;
+
+public class Patient {
+    private String email;
+    private String password;
+    private long idCardNo;
```

Comment:
Typy numeryczne uzywamy wtedy kiedy jest sens wykonywac np. na tych polach jakies operacje artytmetyczne (dodawanie, odejmowanie, mnozenie, dzielenie). 

Więc idCardNo nie powinien byc typem numerycznym tylko Stringiem

Reply:
Zwracajmy uwagę czy używamy małych typów prymitywnych czy referencyjnych (obiektowych).

Jaki używamy typów prymitywnych nie mogą one przyjąć wartości null -> w sytuacji kiedy taka wartośc nie zostanie przesłana to przyjmie wartośc defaultowa np. dla inta = 0. 

Może to spowodować, że w sytuacji kiedy użytkownik nie poda takiej danej to automatycznie nasz kod przyjmie ze wartosc = 0 -> co moze miec katastrofalne skutki np mamy aplikacje odpowiedzialna za przetwarzanie zamowien.
Ktos w modelu zamówienia podał cene jako double -> ktos wysyla zapytanie o utworzenie nowego zamowienia nie podaje ceny i my za darmo wysylamy mu produkty.

Reply:
W sytuacji kiedy to uzytkownik przesyła dane zawsze powinnismy sie zastanowic czy taka dana moze byc pusta -> nie zostac wyslana i swiadomie uzyc np typu Integer